core/container.go

```diff
@@ -835,6 +835,23 @@ func (container *Container) Exec(ctx context.Context, gw bkgw.Client, defaultPla
 }
 
 func (container *Container) ExitCode(ctx context.Context, gw bkgw.Client) (*int, error) {
+	content, err := container.MetaFileContents(ctx, gw, "exitCode")
+	if err != nil {
+		return nil, err
+	}
+	if content == nil {
+		return nil, nil
+	}
+
+	exitCode, err := strconv.Atoi(*content)
+	if err != nil {
+		return nil, err
+	}
+
+	return &exitCode, nil
+}
+
+func (container *Container) MetaFileContents(ctx context.Context, gw bkgw.Client, filePath string) (*string, error) {
 	file, err := container.MetaFile(ctx, gw, "exitCode")
```

Comment:
`"exitCode"` -> `filePath` I think, should fix the test failures

Reply:
Thanks!